### PR TITLE
Re-enable and fix acceptance tests. (Don't manage `/var/run/redis` on Debian systems)

### PR DESCRIPTION
**Pull request by alexjfisher**

### CODE REVIEW

File: .sync.yml [hunk 1]
```diff
@@ -1,8 +1,15 @@
 ---
+.travis.yml:
+  docker_sets:
+    - set: ubuntu1404-64m
+    - set: ubuntu1604-64m
+    - set: centos6-64m
+    - set: centos7-64m
+    - set: debian8-64m
```
Comment (bastelfreak) [inline .sync.yml]: why is there an `m` at the end of each line?

File: manifests/config.pp [hunk 2]
```diff
@@ -45,28 +45,12 @@
...
-      }
-    }
-
+  unless $facts['osfamily'] == 'Debian' or $service_provider_lookup == 'systemd' {
     file { '/var/run/redis':
       ensure => 'directory',
       owner  => $::redis::config_owner,
-      group  => $var_run_redis_group,
-      mode   => $var_run_redis_mode,
+      group  => $::redis::config_group,
```
Comment (bastelfreak) [inline manifests/config.pp]: latest modulesync doesn't allow `::` after `$`, so you might want to directly remove it.

Comment (ekohl) [reply manifests/config.pp]: IMHO not. The module is filled with it and this change keeps it consistent.

Comment (alexjfisher) [reply manifests/config.pp]: That's only actually for classes.  I think there'll be a separate PR where the variables are sorted out.

File: spec/acceptance/nodesets/archlinux-2-x64.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (ekohl) [inline spec/acceptance/nodesets/archlinux-2-x64.yml]: Would it be an idea to add these nodesets as unmanaged? Longer term I'd like to remove them and if this module isn't using them, let's not pollute its history.

File: spec/acceptance/suites/default/redis_sentinel_one_node_spec.rb [hunk 4]
```diff
@@ -9,6 +9,13 @@
                  'redis'
                end
 
+  # We're testing with `manage_repo` true.  In redis-sentinel 5, the daemon has its own rundir
+  pidfile = if fact('operatingsystem') == 'Ubuntu' && fact('operatingsystemmajrelease') == '16.04'
```
Comment (ekohl) [inline spec/acceptance/suites/default/redis_sentinel_one_node_spec.rb]: Could you implement this in pure puppet below?

File: .sync.yml (hunk 1 above)
Comment (alexjfisher) [reply .sync.yml]: Role `master` was needed for one of the beaker helpers. `run_task` IIRC.

File: spec/acceptance/suites/default/redis_sentinel_one_node_spec.rb (hunk 4 above)
Comment (alexjfisher) [reply spec/acceptance/suites/default/redis_sentinel_one_node_spec.rb]: I suppose I may as well.  Ideally, the module itself would set the default correctly, but that's for another day.

File: spec/acceptance/nodesets/archlinux-2-x64.yml (hunk 3 above)
Comment (alexjfisher) [reply spec/acceptance/nodesets/archlinux-2-x64.yml]: Sure.  Can you remind me of the correct syntax for `.sync.yml`?

**Comment by alexjfisher** [comment]
https://github.com/voxpupuli/puppet-redis/pull/299/commits/1e3297e64384339df11a84b385df197a8983423a basically reverts https://github.com/voxpupuli/puppet-redis/pull/173

> rundir directories are 2775

On my system, `/var/run/postgres` was the *only* rundir with that permission.  I went through the distribution redis packages and ppa packages for Debian 7,8,9 and Ubuntu 14.04, 16.04.  They *all* use 0755 (via their init scripts).

File: spec/acceptance/nodesets/archlinux-2-x64.yml (hunk 3 above)
Comment (ekohl) [reply spec/acceptance/nodesets/archlinux-2-x64.yml]: https://github.com/voxpupuli/puppet-windows_env/blob/542466bc58552befc8b209f0b317910c515cc9b3/.sync.yml#L4-L13

File: spec/acceptance/suites/default/redis_sentinel_one_node_spec.rb (hunk 4 above)
Comment (alexjfisher) [reply spec/acceptance/suites/default/redis_sentinel_one_node_spec.rb]: Done (assuming I've not messed it up).

Comment (alexjfisher) [reply spec/acceptance/suites/default/redis_sentinel_one_node_spec.rb]: I messed it up...

**Pull request merged**